pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -406,7 +464,9 @@ func (c *Cluster) provisionPodTemplateSpec(devices []rookalpha.Device, selection
 	if c.HostNetwork {
 		podSpec.DNSPolicy = v1.DNSClusterFirstWithHostNet
 	}
-	c.placement.ApplyToPodSpec(&podSpec)
+	if osdObject.pvc.ClaimName == "" {
+		c.placement.ApplyToPodSpec(&podSpec)
```

Comment:
When the StorageClassDeviceSet is merged with this, we will need to apply its placement here.

Reply:
Yes.
@ashishranjan738 could you update this part of code.

Reply:
yup done